cluster/ubuntu/util.sh

```diff
@@ -373,6 +372,7 @@ function detect-nodes() {
 
 # Instantiate a kubernetes cluster on ubuntu
 function kube-up() {
+  source "${KUBE_ROOT}/cluster/common.sh"
```

Comment:
kube-up and kube-down are "high level functions" ... they call helper functions like provision-master (which is specific to this file), but then call functions from common.sh (e.g. verify-cluster, create-kubeconfig) which are not ubuntu specific.

Also, this code was this way before commit 39a25edb71f6ab4952a20af11d7f8fadd640347c